### DISCUSSION TITLE
[A-L] (2023/24) Foglio 5 - Esercizio 10

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#312] [Esercizi - Viaggi/Piazza]
![Screenshot 2023-11-04 181439](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/b58385ee-35b2-4c57-94d7-32221874d577)

**Comment by IonutCicio** [comment]
Sia $(G, \cdot)$ un gruppo e $\rho$ una relazione di equivalenza, per cui
- $\forall g \in G, g \rho g$
- $\forall g, h \in G, g\rho h \implies h \rho g$
- $\forall g, h, i \in G, g \rho h, h \rho i \implies g \rho i$

Diremo che $\rho$ è compatibile con $\cdot$ se $g\rho g', \gamma \rho \gamma' \implies (g \cdot \gamma) \rho (g' \cdot \gamma')$, per cui il loro prodotto è ben posto rispetto alla relazione di equivalenza.

Dimostrare che se $\rho$ è compatibile con $\cdot$ allora $(G/\rho, \star)$ ha una naturale struttura di gruppo data da:

$$
[g] \star [h] := [g \cdot h]
$$

#### $\star$ è ben definita?

Prendiamo $g, g' \in G/\rho \mid g \rho g'$ e $\gamma, \gamma' \in G/\rho \mid \gamma \rho \gamma'$, vogliamo dimostrare che prendendo due rappresentanti diversi, il prodotto si mantiene nella stessa classe, per cui $(g \star \gamma) \\ \rho \\ (g' \star \gamma')$. Possiamo riscrivere la formula precedente come $(g \cdot \gamma) \\ \rho \\ (g' \cdot \gamma')$ per come è definita l'operazione $\star$. Ma sappiamo che $\rho$ è compatibile con $\cdot$ quindi l'affermazione precedente è vera per definizione di "compatibile", quindi $\star$ è ben definita.

#### $(G/\rho, \star)$ è un gruppo?

Bisogna verificare che:
- $\forall g, h, i \in G/\rho, (g \star h) \star i = g \star (h \star i)$
- $\exists e\_{\rho} \in G/\rho \mid \forall g \in G/\rho, g \star e\_{\rho} = g = e\_{\rho} \star g$
- $\forall g \in G/\rho, \exists g^{-1} \in G/\rho \mid g \star g^{-1} = e\_{\rho} = g^{-1} \star g$

1. $g, h, i \in G/\rho, ([g] \star [h]) \star [i] = (g \cdot h) \cdot i =$ (sapendo che $G$ è un gruppo, quindi $\cdot$ è associativa) $=  g \cdot (h \cdot i) = [g] \star ([h] \star [i])$
2. Si consideri $e\_{\rho} \in G/\rho, e\_{\rho} = [e], e \in G$, quindi $e\_{\rho}$ è la classe di equivalenza rispetto a $\rho$ dell'elemento neutro in $G$. Quindi, se prendiamo $[g] \in G$, $[g] \star [e] = [g \cdot e] = [g] = [e \cdot g] = [e] \star [g]$, usando il fatto che $G$ è un gruppo.
3. Si considerino $g, g^{-1} \in G$, allora $[g] \star [g^{-1}] = [g \cdot g^{-1}] = [e] = [g^{-1} \cdot g] = [g^{-1}] \star [g]$ usando il fatto che $G$ è un gruppo e che $[e] = e\_{\rho}$ l'elemento neutro in $G/\rho$

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/e9822bcf-aa62-4fea-9717-3ac59f2b3b0e)
